### ISSUE TITLE
When switching off menu effects, panel effects are switched off too

### ISSUE THREAD

**Issue by vassilisw** [linuxmint/Cinnamon#7264]
```
 * Cinnamon version - 3.2 3.4 3.6
 * Distribution - Mint 18.1 18.2 18.3
 ```

**Issue**
When switching off the menu effects (for menu bar, right click menu, etc) from preferences, the panel's applets popup window effects (when clicking the volume applet for example) are gone too!

**Steps to reproduce**
Just switch off the menu effects from the preferences and click on panel applets (volume, calendar, etc).

**Expected behaviour**
As of cinnamon 2.6, 2.8 in LM 17.2 and 17.3, menu-bar effects state (on/off) does not affect the panel's applet effects. This is the right behaviour. Menu-bars and contextual menus are _totally separated from the cinnamon and the cinnamon panel_. The menu effects can be annoying and destructive but the panel applet's effects are enjoyable and good looking.

**Comment by JosephMcc** [comment]
This isn't a bug. It's intentional behavior.

**Comment by vassilisw** [comment]
The first guy closed it because instead of 3.2 3.4 and 3.6 I mentioned **3+** just for the second guy to close it **after 10 minutes** too, saying that this is intentional. This reminds me the army.
Anyway... It could be considered as a "Feature request" or "Suggestion" as many others here. Leaving 
 not being able to explain the behaviour and discouraged of being helpful.

**Comment by JosephMcc** [comment]
Fair enough but even as a feature request it's going to be closed. We are no longer just leaving feature requests open here forever. We close them and if they are something we want to change implement we add them to our roadmap of things to do.
